core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/RuntimeRoutesTestJsonCodec.scala

```diff
@@ -81,7 +81,7 @@ object RuntimeRoutesTestJsonCodec {
     "startUserScriptUri",
     "runtimeConfig",
     "userJupyterExtensionConfig",
-    "autopause",
+    "autopauseEnabled",
```

Comment:
this will be breaking change for clients. Terra UI isn't the only client for Leonardo. So I'd prefer we keep the previous name